pkg/kubectl/cmd/cp.go

```diff
@@ -75,38 +75,41 @@ func NewCmdCp(f cmdutil.Factory, cmdOut, cmdErr io.Writer) *cobra.Command {
 }
 
 type fileSpec struct {
-	PodNamespace string
-	PodName      string
-	File         string
+	PodNamespace    string
+	PodName         string
+	File            string
+	LocalFileExists bool // indicates if File exists in local filesystem
```

Comment:
Since `LocalFileExists` is not a MUST for `fileSpec`, I suggest keep it as it is.

We can leave it in `runCopy` to handle this.